<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomoCalcWidget</class>
 <widget class="QWidget" name="HomoCalcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Ref_Point">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>点任务选择</string>
   </property>
   <widget class="QComboBox" name="comboBoxPoint">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点任务：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点结果:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPointResult">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Mode">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Homograph参数模式</string>
   </property>
   <widget class="QGroupBox" name="groupBox_DefaultValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>错误时默认数据</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>像素坐标X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPixX">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>像素坐标Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPixY">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>物理坐标X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPhyX">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>物理坐标Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPhyY">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>动态homo参数索引:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtHomoPara">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButton_homo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>使用指定的homograph参数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Grst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>使用GRst里面动态参数</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_OffsetCacl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>偏移计算</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_rotate_angle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>旋转弧度选择</string>
   </property>
   <widget class="QComboBox" name="comboBoxAngle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>弧度任务：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>弧度结果:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAngleResult">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
